modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/DebugCacheVersionConflictResolverImpl.java

```diff
@@ -63,6 +63,16 @@ public DebugCacheVersionConflictResolverImpl(byte clusterId, String conflictReso
         return res;
     }
 
+    /** @return Conflict resolve field value, or specified {@code val} if the field not found. */
+    private Object debugValue(Object val) {
+        try {
+            return super.value(val);
+        }
+        catch (Exception e) {
+            return val;
```

Comment:
LGTM in general, Just a minor comment.

can we add java comment about logging this exception - it will be logged at a top level?